### ISSUE TITLE
Support graphical visualization for spatial types in query results like SSMS

### ISSUE THREAD

**Issue by sanagama** [Microsoft/vscode-mssql#825]
- MSSQL Extension Version: 0.3.0
- VSCode Version: 1.11.2
- OS Version: macOS Sierra 10.12.4

Steps to Reproduce:

Type the following query in the SQL editor and execute against SQL Server (I tried with SQL 2016):
DECLARE @g geometry;
SET @g = geometry::STGeomFromText('CIRCULARSTRING(2 0, 1 1, 0 0)', 0);
select @g

Result: the results window shows a column with binary data
Desired/Expected: graphical visualization for geometry/geography data in results window, similar to SSMS

**Comment by kburtram** [comment]
Marking this is blocked since we need SQLCLR support in .Net Core before we can implement this feature.

**Comment by kburtram** [comment]
Thanks for the suggestion. Closing as out of scope; graphical visualization of geospatial data isn't planned for this extension.
